addon/components/au-modal.hbs

```diff
@@ -43,15 +43,18 @@
             {{@title}}
           {{/if}}
         </h1>
-        <button
-          class="au-c-modal__close"
-          type="button"
-          data-test-modal-close
-          {{on "click" this.closeModal}}
-        >
-          <AuIcon @icon="cross" @size="large" />
-          <span class="au-u-hidden-visually">Venster sluiten</span>
-        </button>
+        {{#if this.closeTriggers.x}}
```

Comment:
I'm not entirely sure if making it possible to remove the close button (as a trigger) in the modal header is good practice accessibility/UX wise?